tests/dotnet-e2e/fileops/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿var TEXT = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.";
+
+// Read
+
+{
+    var dat = File.ReadAllText("/app/test.txt");
+
+    if (dat != TEXT)
+        throw new Exception($"Expected {TEXT}, got {dat}");
+}
+
+
+// Write
+
+{
+    var filePath = createTempFile();
+    checkFileExistsOnHost(filePath);
+
+    var dat = File.ReadAllText(filePath);
+
+    if (dat != TEXT)
+        throw new Exception($"Expected {TEXT}, got {dat}");
+
+    File.Delete(filePath);
+}
+
+
+// Lseek
+
+{
+    var filePath = createTempFile();
+    checkFileExistsOnHost(filePath);
+
+    var fileStream = File.Open(filePath, FileMode.Open);
+    var buf = new byte[TEXT.Length];
+
+    // https://learn.microsoft.com/en-us/dotnet/api/system.io.filestream.canseek?view=net-6.0
+    if (!fileStream.CanSeek)
+        throw new Exception("file stream CanSeek failure");

```

Comment:
Do we really need this check?

Reply:
I was following the Go tests line-for-line. Perhaps too closely! Upon review, I don't think this check adds value to the Lseek code block in this file.

Removed 👍🏻